app/lib/index.ts

```diff
@@ -39,7 +39,6 @@ const argv = parseArgs(process.argv, process.cwd())
 
 if (!app.requestSingleInstanceLock()) {
     app.quit()
-    process.exit(0)
```

Comment:
Looking at all documentation for app.requestSingleInstanceLock() everything said to use app.quit(). app.quit() calls beforeunload and unload event handlers before exiting. Doesn't look like any of those handlers are in use right now but they may be added in the future.

I just felt that the process.exit(0) was redundant after calling app.quit().

I couldn't find much documentation on calling process.exit in an electron application other than `process.exit is a Node API, so I would assume that it may not do all the right things with regard to an Electron application.`

I'll happily add it in if app.quit() isn't enough!

Reply:
Maybe app.exit(0) is a good solution?